samples/pmic/native/npm2100_fuel_gauge/README.rst

```diff
@@ -0,0 +1,157 @@
+.. _npm2100_fuel_gauge:
+
+nPM2100: Fuel gauge
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The fuel gauge sample demonstrates how to calculate the state of charge of a supported primary cell battery using `nPM2100 <nPM2100 product website_>`_ and the :ref:`nrfxlib:nrf_fuel_gauge`.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM2100 evaluation kit that you need to connect to the development kit as described in `Wiring`_.
+
+Overview
+********
+
+This sample allows the calculation of state of charge from a battery connected to the nPM2100 PMIC.
+
+Battery models for Alkaline AA, AA, LR44, and Lithium-manganese dioxide coin cell CR2032 batteries are included. The active battery model can be changed using a shell command, as illustrated in `Testing`_.
+
+Wiring
+******
+
+With this configuration, the nPM2100 EK is wired to supply power to the DK.
+This ensures that the TWI communication is at compatible voltage levels, and represents a realistic use case for the nPM2100 PMIC.
+
+.. note::
+
+   In order prevent leakage currents and to continue being able to program the DK, do not remove the DK USB connection.
+
+.. note::
+
+   Unplug the battery from nPM2100 EK, and set the DK power switch to "OFF" while applying the wiring described below.
+   In case of issues communicating with or programming the DK after applying the wiring, try to power cycle the DK and EK.
```

Comment:
```suggestion
   Unplug the battery from the nPM2100 EK and set the DK power switch to "OFF" while applying the wiring described below.
   If you have issues communicating with the DK or programming it after applying the wiring, try to power cycle the DK and EK.
```

Reply:
How about combining the two consecutive notes to one?

Reply:
I'm fine with that, if you think it is better from a style perspective. The reason for having them separate is that they cover two slightly different things. But I can have two small paragraphs in the same note